balloc/balloc.h

```diff
@@ -213,6 +213,23 @@ enum {
 	M0_BALLOC_BUDDY_LOOKUP_MAX = 10,
 };
 
+/*
```

Comment:
Why is it moved here?

Reply:
No sure, why it is showing in my diff, i have not moved, just tuned BALLOC_DEF_BLOCKS_PER_GROUP and one modified api m0_balloc_create()